Scenes/MainScenes/Game.cs

```diff
@@ -826,4 +844,46 @@ void deleteChildren(Node theNode)
     }
   }
 
+  void _on_go_to_game(bool isAuthorized)
+  {
+    if (isAuthorized == false)
+    {
+      _serilogger.Information("User no authenticated,retry");
+      loginScreen.GetNode<TextureRect>("AuthLoadingRect").Visible = false;
+
+      return;
+    }
+
+    loginScreen.GetNode<TextureRect>("NoAuthorizedRect").Visible = false;
+    loginScreen.GetNode<TextureRect>("AuthLoadingRect").Visible = false;
+
+    this.myUuid = authorization.getUserId();
+    this.goToTheGame(myUuid);
```

Comment:
'_on_go_to_game(bool isAuthorized)'

This is happen when the auth process is completed.
Because the uuid is calculated from a token that I saved in the auth process, i created a function to calculate it there.